.github/workflows/dockerimage.yml

```diff
@@ -18,4 +19,4 @@ jobs:
     - uses: actions/checkout@v2
     - uses: technote-space/get-diff-action@v1
     - name: Get the list of modified files only
-      run: docker build -t shellspec . && docker run -v $PWD:/app shellspec bash -c "cd /app && ./test.sh ${{ env.GIT_DIFF }}"
+      run: pip install toml && python generate.py ${{ env.GIT_DIFF }} && docker build -t shellspec . && docker run -v $PWD:/app shellspec bash -c "cd /app && ./test.sh ${{ env.GIT_DIFF }}"
```

Comment:
Let's take a task to build the Docker image separately and store it in a Docker image repository rather building it for each PR.